src/demo.html

```diff
@@ -100,6 +100,7 @@ <h1>MicroPython-micro:bit simulator example embedding</h1>
               </option>
               <option value="sound_effects_user">Sound effects (user)</option>
               <option value="speech">Speech</option>
+              <option value="stack_size">Stack size</option>
```

Comment:
What does this do?

Reply:
Adds the example to the list in [the demo](https://review-python-simulator.usermbit.org/asyncify-stack-size/demo.html)

![image](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/assets/44397098/ff15d21d-9c4f-4d31-bd82-669604389949)

Reply:
Ah, very nice!